packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -366,8 +535,35 @@ export const outOfOfficeEntriesList = async ({ ctx }: { ctx: { user: NonNullable
         },
       },
       notes: true,
+      user: fetchTeamMembersEntries
+        ? {
+            select: {
+              id: true,
+              username: true,
+              email: true,
+              avatarUrl: true,
+            },
+          }
+        : false,
+    },
+    orderBy: {
+      end: "desc",
```

Comment:
Imo it is better to sort by start date rather than end.

Reply:
updated